cam_intrinsic: 
    [ 762.7249337622711, 0.0, 640.5,
     0.0, 762.7249337622711,360.5, 
     0.0, 0.0, 1.0]
velo_to_cam:
    [ 0, -1, 0, 0,
      0, 0, -1, -0.1,
      1, 0, 0, -0.4,
      0, 0, 0, 1]
image_topic: "/realsense/image_raw"
lidar_topic: "/point_no_ground"
bbox_topic: "/bounding_boxes"
w1: 10
w2: 5
w3: 20